btb/selection/selector.py

```diff
@@ -11,7 +11,7 @@ def __init__(self, choices, **kwargs):
         """
         self.choices = choices
 
-    def compute_rewards(self, scores):
+    def compute_rewards(saelf, scores):
```

Comment:
There's a type in the `saelf` argument!

Also, this and all the other models that do not use the `self` argument could be converted to `@staticmethod`s